build_projects/shared-build-targets-utils/Utils/Monikers.cs

```diff
@@ -38,7 +38,7 @@ public static string GetProductMoniker(BuildTargetContext c, string artifactPref
             string rid = Environment.GetEnvironmentVariable("TARGETRID") ?? RuntimeEnvironment.GetRuntimeIdentifier();
 
             // Look for expected RIDs, including Portable one, for Linux
-            if (rid.StartsWith("linux-") || rid == "ubuntu.16.04-x64" || rid == "ubuntu.16.10-x64" || rid == "fedora.23-x64" || rid == "fedora.24-x64" || rid == "opensuse.13.2-x64" || rid == "opensuse.42.1-x64" || rid == "debian.8-armel" || rid == "tizen.4.0.0-armel")
+            if (rid.StartsWith("linux-") || rid == "ubuntu.16.04-x64" || rid == "ubuntu.16.04-arm" || rid == "ubuntu.16.10-x64" || rid == "fedora.23-x64" || rid == "fedora.24-x64" || rid == "opensuse.13.2-x64" || rid == "opensuse.42.1-x64" || rid == "debian.8-armel" || rid == "tizen.4.0.0-armel")
```

Comment:
@hqueue It is simply to maintain status quo and avoid unrequired additional changes for the moment that we can do without for cross build bringup. Over a long period, our build infrastructure is going to change for Core-Setup significantly that this piece of code will just disappear and will no longer be relevant.